include/deal.II/fe/fe_interface_values.h

```diff
@@ -1471,20 +1488,43 @@ class FEInterfaceValues
          const unsigned int              sub_face_no_neighbor);
 
   /**
-   * Re-initialize this object to be used on an interface given by a single face
+   * Same as above, but should be used when dealing with different
+   * FiniteElements on the two side of a facet.
+   */
+  template <class CellIteratorType, class CellNeighborIteratorType>
+  void
+  reinit_hp(const CellIteratorType &        cell,
+            const unsigned int              face_no,
+            const unsigned int              sub_face_no,
+            const CellNeighborIteratorType &cell_neighbor,
+            const unsigned int              face_no_neighbor,
+            const unsigned int              sub_face_no_neighbor);
+
+  /**
+   * Re-initialize this object to be used on an interface
+   * given by a single face
    * @p face_no of the cell @p cell. This is useful to use FEInterfaceValues
    * on boundaries of the domain.
    *
-   * As a consequence, members like jump() will assume a value of zero for the
-   * values on the "other" side. Note that no sub_face_number is needed as a
-   * boundary face can not neighbor a finer cell.
+   * As a consequence, members like jump() will assume a value
+   * of zero for the values on the "other" side. Note that no
+   * sub_face_number is needed as a boundary face can not
+   * neighbor a finer cell.
    *
-   * After calling this function at_boundary() will return true.
+   * After calling this function at_boundary() will return
+   * true.
```

Comment:
There are a lot of unnecessary changes to the documentation. Did you reindent these files with the same version of clang-format that we typically use?

Reply:
Yes, I'm using the same one. I can't understand what's going on.

Reply:
Yes, I'm using the same one. I can't understand why it's formatting in that way.